src/Features/CSharp/Portable/ExtractMethod/CSharpSelectionResult.ExpressionResult.cs

```diff
@@ -24,10 +24,9 @@ internal abstract partial class CSharpSelectionResult
         private sealed class ExpressionResult(
             SemanticDocument document,
             SelectionType selectionType,
-            TextSpan originalSpan,
```

Comment:
remove this.  this should only be the starting info from the user.  we don't want to pass this along as it's actively confusing as to when you'd use this over 'final span'.